docs/application/native/guides/multimedia/image-classification.md

```diff
@@ -0,0 +1,216 @@
+# Image Classification
+
+You can classify an image, which belongs to the corresponding labels.
+
+The main features of the Media Vision Inference API include:
+
+- Classifying an image
```

Comment:
Give a line break after  

- Classifying an image

 Like this 
The main features of the Media Vision Inference API include:

- Classifying an image: 

Image classification is one of the main features of the Media Vision Inference API. You can [classify](#classify) an image, which matches to its corresponding labels. For example, to classify an image consisting of a food item, you can apply a food classification model while doing inference of the decoded image data.